packages/client/tests/functional/metrics/enabled/tests.ts

```diff
@@ -269,7 +269,9 @@ testMatrix.setupTestSuite(
             {
               key: 'prisma_pool_connections_idle',
               labels: {},
-              value: expect.any(Number),
+              // This can sometimes be reported as 1 or 0,
+              // as metrics are reported asynchronously.
+              value: 0 || 1,
```

Comment:
That will always evaluate to 1, so if the actual value would be 0 test won't pass.
It does not seem like there is built-in asymetric matcher for `either`, so it looks like we need to write our own, unfortunately.

Reply:
True 😅 I also didn't see a built in but I'm also not sure is it will actually be flaky, it was reliably `1` when running locally

Reply:
I pushed a fix